common/src/primitives/per_thousand.rs

```diff
@@ -57,6 +57,13 @@ impl PerThousand {
         let result = Self::new(value)?;
         Some(result)
     }
+
+    pub fn into_decimal_str(&self) -> String {
+        format!(
+            "{}%",
+            Amount::from_atoms(self.0.into()).into_fixedpoint_str(1)
+        )
+    }
```

Comment:
Would it make sense to put this into a `Display` trait impl?

Reply:
It could be both a percentage and decimal. I wouldn't use Display.